doc/sphinx/system_setup.rst

```diff
@@ -210,6 +210,15 @@ the temperature of a thermostat, you actually do not define the
 temperature, but the value of the thermal energy :math:`k_B T` in the
 current unit system (see the discussion on units, Section :ref:`On units`).
 
+All thermostats have a ``seed`` argument that controls the state of the random
+number generator (Philox Counter-based RNG). This seed is required on first
+activation of a thermostat, unless stated otherwise. It can be omitted in
+subsequent calls of the method that activates the same thermostat. The random
+sequence also depends on a monotonically increasing global counter that is
+incremented after each integration step. It is the user's responsibility to
+decide whether the thermostats should be deterministic (by using a fixed seed)
```

Comment:
that's confusing... the thermostat is always deterministic for a given seed.

Reply:
I find it confusing too, it's probably better to simply remove that sentence